server/routes/controllers/page-loader.ts

```diff
@@ -49,6 +50,24 @@ export class PageLoader {
     }
   }
 
+  static buildItemRoutes(items, router: Router) {
```

Comment:
really tiny nitpick for future readability

```suggestion
  static buildInventoryItemRoutes(items, router: Router) {
```